### PR TITLE
[11.0] [MIG] users ldap mail

### PR DESCRIPTION
**Pull request by etobella**
Migration to 11.0.
Same code that existed on #1 and OCA/server-tools#1018.

### CODE REVIEW

**Review by dreispt — APPROVED** [review]

File: users_ldap_mail/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     'name': "LDAP mapping for user name and e-mail",
-    'version': "10.0.1.0.0",
+    'version': "11.0.1.0.0",
     'depends': ["auth_ldap"],
     'author': "Daniel Reis (https://launchpad.com/~dreis-pt),"
```
Comment (lasley) [inline users_ldap_mail/__manifest__.py]: @dreispt - are you good with removing your launchpad from this?

File: users_ldap_mail/views/ldap_installer_views.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```
Comment (lasley) [inline users_ldap_mail/views/ldap_installer_views.xml]: Please align file name with [contribution guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming)

`res_company_ldap_view.xml`

**Review by lasley — APPROVED** [review]
Minor comments. Thanks!

File: users_ldap_mail/__manifest__.py (hunk 1 above)
Comment (dreispt) [reply users_ldap_mail/__manifest__.py]: Yes, of course 😄

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
This one is ready to merge.

**Comment by lasley** [comment]
I think this is the first v11 I've merged. Merging this many commits is intimidating.

**Comment by lasley** [comment]
Btw thanks @etobella! 😉

**Comment by dreispt** [comment]
@lasley Not really: only the two last commits are new; the rest is just module history.

**Comment by pedrobaeza** [comment]
@lasley check https://odoo-community.org/groups/contributors-15/contributors-64315?mode=thread&date_begin=&date_end= for the explanation

**Pull request merged**